docs/en/samples/multi_dataset_same_node_accessing.md

```diff
@@ -45,7 +45,7 @@ spec:
               operator: In
               values:
                 - "multi-dataset"
-
+  placement: "Shared" // set Exclusive or empty means dataset exclusive
```

Comment:
What about not setting placement?

Reply:
exclusive mode?

Reply:
Yes, I think we need to clarify it to the end user.